chrome/background.js

```diff
@@ -51,4 +79,27 @@ function onMessage(request, sender, sendResponse) {
     // Need to return true if we are planning to sendResponse asynchronously
     return true;
   }
+
+  if (request.action == "dismissOTP" && sender.tab.id in tabInfos) {
+    delete tabInfos[sender.tab.id];
+  }
+}
+
+function onTabUpdated(tabId, changeInfo, tab) {
+  if (changeInfo.url) {
+    if (getHostname(changeInfo.url) != tabInfos[tabId].hostname) {
```

Comment:
I've got an error on this line in the console: `Error in event handler for tabs.onUpdated: TypeError: Cannot read property 'hostname' of undefined`